presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/BaseElasticsearchSmokeTest.java

```diff
@@ -840,6 +892,12 @@ private void addAlias(String index, String alias)
 
     protected abstract String indexMapping(@Language("JSON") String properties);
 
+    private void createIndex(String indexName)
+            throws IOException
+    {
+        client.getLowLevelClient().performRequest("PUT", "/" + indexName, ImmutableMap.of());
```

Comment:
is it equiv to `createIndex(indexName, '{}')`?

Reply:
No, `ElasticsearchClient` expects at least one property to be present when processing mappings.